annotation/tests/test_jobs.py

```diff
@@ -88,3 +184,196 @@ def test_check_annotators(
 ):
     with pytest.raises(FieldConstraintError):
         check_annotators(annotators, validation_type)
+
+
+def test_collect_job_names_all_db():
+    mock_session = MagicMock()
+    mock_job = namedtuple("mock_job", ["job_id", "name"])
+    mock_session.query().filter().all.return_value = [
+        mock_job(1, "test1"),
+        mock_job(2, "test2"),
+    ]
+    expected_result = {1: "test1", 2: "test2"}
```

Comment:
Maybe make this test setup a fixture, since it appears in this and in the next test

Reply:
You are right there was no need to use in both tests mock_job